IM_calculation/source_site_dist/src_site_dist.py

```diff
@@ -34,15 +28,18 @@ def calc_rrup_rjb(srf_points: np.ndarray, locations: np.ndarray):
     rjb : np.ndarray
         The rjb distance for the locations, shape/order same as locations
     """
-    rrups = np.empty(locations.shape[0])
-    rjb = np.empty(locations.shape[0])
+    rrups = np.empty(locations.shape[0], dtype=np.float32)
+    rjb = np.empty(locations.shape[0], dtype=np.float32)
+
+    srf_points = srf_points.astype(np.float32)
+    locations = locations.astype(np.float32)
 
     # Size limit for memory capacity
-    size = 5000
+    size = 1000
```

Comment:
I think this should either be defined as a constant (uppercase) right at the top of the function, or make it a parameter of the function. I prefer the latter approach. Also maybe a more descriptive name, such as `n_stations_per_iter`

Reply:
A function parameter makes a lot of sense for this. I didn't like it but thought a constant was a bit overkill.